### PR TITLE
Mirror IE + Chrome -> Edge for api/ (G-M)

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR is a part of #5214, in mirroring Chromium data over to Edge as per the new Edge version.  This performs four tasks: setting the existing `true` values to `≤18`, mirroring the Chromium data to any entires marked as `null` whilst setting the version to `≤79`, mirroring the Chromium data to entries marked as `false`, and setting anything that's `true` in IE to `12` in Edge.  The majority of this data has been generated by the mirroring script, however due to the nature of the migration, there's also a good portion modified manually (I do plan on upgrading the mirroring script to work better for this).

### CODE REVIEW

File: api/MediaElementAudioSourceNode.json [hunk 1]
```diff
@@ -61,7 +61,8 @@
               "notes": "Before Chrome 59, the default values were not supported."
             },
             "edge": {
-              "version_added": null
+              "version_added": "≤79",
+              "notes": "Before Edge 59, the default values were not supported."
```
Comment (Elchi3) [inline api/MediaElementAudioSourceNode.json]: Please remove this note.

File: api/MediaStreamAudioDestinationNode.json [hunk 2]
```diff
@@ -61,7 +61,8 @@
               "notes": "Before Chrome 59, the default values were not supported."
             },
             "edge": {
-              "version_added": null
+              "version_added": "≤79",
+              "notes": "Before Edge 59, the default values were not supported."
```
Comment (Elchi3) [inline api/MediaStreamAudioDestinationNode.json]: Please remove this note.

File: api/MediaStreamAudioSourceNode.json [hunk 3]
```diff
@@ -61,7 +61,8 @@
               "notes": "Before Chrome 59, the default values were not supported."
             },
             "edge": {
-              "version_added": null
+              "version_added": "≤79",
+              "notes": "Before Edge 59, the default values were not supported."
```
Comment (Elchi3) [inline api/MediaStreamAudioSourceNode.json]: Please remove this note.

**Comment by precia-bbc** [comment]
Two of the `notes` in this PR contain artifacts of a bulk rename from Chrome to Edge:
- "Edge OS" `api/MediaDevies.json L264`
- "Edge Platform Status" hyperlinks to a chrome website ` api/HTMLIFrameElement.json L480`

**Comment by vinyldarkscratch** [comment]
@precia-bbc, since this has already been merged, can you open up a new issue or pull request?

**Pull request merged**